src/main/java/com/puppycrawl/tools/checkstyle/checks/regexp/RegexpSinglelineCheck.java

```diff
@@ -94,7 +108,60 @@
  *   &lt;property name="maximum" value="10"/&gt;
  * &lt;/module&gt;
  * </pre>
- *
+ * <p>Example:</p>
+ * <pre>
+ * &#47;**
+ * * This file is copyrighted under CC. // Ok, as the file contains a copyright statement.
+ * *&#47;
+ * class MyClass {
+ *      private String myString;
+ *      private int myInt;
+ * }
+ * </pre>
+ * <pre>
+ * &#47;** // violation, as the file doesn't contain a copyright statement.
+ * * MyClass as a configuration example.
+ * *&#47;
+ * class MyClass {
+ *      private String myString;
+ *      private int myInt;
+ * }
+ * </pre>
+ * <p>
+ *  An example of how to configure the check to make sure java files contains the term 'license'.
+ * </p>
+ * <pre>
+ * &lt;module name="RegexpSingleline"&gt;
+ *     &lt;property name="format" value="license"/&gt;
+ *     &lt;property name="minimum" value="1"/&gt;
+ *     &lt;property name="maximum" value="10"/&gt;
+ *     &lt;property name="ignoreCase" value="true"/&gt;
+ *     &lt;!--  Configure a message to be shown on violation of the Check. --&gt;
+ *     &lt;property name="message"
+ *           value="File must contain at least one occurrence of 'license' term"/&gt;
+*      &lt;!--  Perform the Check only on files with java extension. --&gt;
+ *     &lt;property name="fileExtensions" value="java"/&gt;
```

Comment:
please make example for non-java file, make it for `*.sql` file for example.

Reply:
okay, I will make the changes soon

Reply:
done